docs/theory/ensemble_based_methods.rst

```diff
@@ -175,25 +177,40 @@ equation for how the boat moves in time, and at selected points in time :math:`t
 
 The purpose of the Kalman Filter is to determine an updated :math:`x_k` from
 :math:`x_{k-1}` and :math:`d_k`. The updated :math:`x_k` is the value that
-*minimizes the variance* :math:`\sigma_k`. The equations for updated position
-and uncertainty are:
+*minimizes the variance* :math:`\sigma_k`. 
+The updated position is given by:
 
 .. math::
 
-   x_k = x_k^{\ast}\frac{\sigma_d^2}{\sigma_k^2 + \sigma_d^2} + x_d
-   \frac{\sigma_k^2}{\sigma_k^2 + \sigma_d^2}
+   x_k = x_k^{\ast}\frac{\sigma_d^2}{\sigma_k^{2*} + \sigma_d^2} + d_k
+   \frac{\sigma_k^{2*}}{\sigma_k^{2*} + \sigma_d^2}
+
+In the equation for the position update, the analyzed position :math:`x_k` is a weighted sum over
+the forecasted position :math:`x_k^{\ast}` and measured position :math:`d_k`.  The weighting
+depends on the relative ratio of the uncertainties :math:`\sigma_k` and :math:`\sigma_d`.
+
+Now let 
+
+.. math:: 
+   
+   K=\frac{\sigma_k^{2*}}{\sigma_k^{2*} + \sigma_d^2}
+
+which allows us to write the udpated position as
+
+.. math:: 
+   x_k = Kd_k + (1 - K)x_k^{\ast} = x_k^{\ast} + K(x_k^{\ast} - d_k)
+
+:math:`K` is the Kalman gain which is a scaling term that chooses a value between the prediction and measurement.
```

Comment:
It took some time for me to understand what "chooses a value between the prediction and measurement" means. But yes, this is good and meaningful. 

Would it be easier to understand if we are more explicit, i.e.
":math:`K` is the Kalman gain which is a weight term, :math: `0<K<1`, that chooses an update value :math: `x_k` that lies between the prediction and measurement."
I prefer "weight" over "scaling", because a weight is typically between 0 and 1, while scaling is to me a (slightly more) general map than a weighting.

If we want to be rigorous, we could add "The K as a weight term holds for the one-dimensional case, but has similar properties in the general sense."

Reply:
I've replaced the current description of `K` with the one you propose.